CAP/gap/DerivedMethods.gi

```diff
@@ -453,22 +453,6 @@ AddDerivationToCAP( ImageEmbedding,
   
   end : CategoryFilter := IsAbelianCategory ); ##FIXME: PreAbelian?
 
-##
```

Comment:
I suggest resurrecting all these derivations with `CanonicalIdentificationFromImageObjectToCoimage` replaced by `InverseMorphismFromCoimageToImageWithGivenObjects`.

@zickgraf: Would you again run in the same problems described in https://github.com/homalg-project/CAP_project/issues/1460?

Reply:
A more refined suggestion is part of this PR https://github.com/homalg-project/CategoricalTowers/pull/377.

Reply:
> I suggest resurrecting all these derivations with `CanonicalIdentificationFromImageObjectToCoimage` replaced by `InverseMorphismFromCoimageToImageWithGivenObjects`.
> 
> @zickgraf: Would you again run into the same problems described in #1460?

I had three problems in https://github.com/homalg-project/CAP_project/issues/1460:

1. The role of `CanonicalIdentificationFromCoimageToImageObject` compared to `MorphismFromCoimageToImage`.
2. The name `CanonicalIdentificationFromCoimageToImageObject` vs. `CanonicalIdentificationFromCoimageObjectToImageObject`.
3. Asymmetry of the derivations using `CanonicalIdentificationFromCoimageToImageObject` with regard to dualization.

The first two points are non-issues for `MorphismFromCoimageToImage`. The third would have to be investigated for `MorphismFromCoimageToImage`. `MorphismFromCoimageToImage` is currently not a CAP operation, as you noticed somewhere else, and the WithGiven version is self-dual, while `CanonicalIdentificationFromCoimageToImageObject` had a proper dual.